tensorboard/webapp/widgets/data_table/header_cell_component.scss

```diff
@@ -47,24 +47,37 @@ $_accent: map-get(mat.get-color-config($tb-theme), accent);
 }
 
 .delete-icon-container {
-  border-radius: 5px;
-  color: mat.get-color-from-palette(mat.$gray-palette, 500);
-  height: 12px;
   margin-left: -12px;
   opacity: 0;
   position: absolute;
-  width: 12px;
 }
 
-.delete-icon-container:hover {
+.delete-icon:hover {
   color: #fff;
 }
 
-.delete-icon-container:hover {
+.delete-icon:hover {
   background-color: mat.get-color-from-palette(mat.$gray-palette, 400);
   cursor: pointer;
 }
 
+button {
```

Comment:
I would rather add this style to the delete icon-class instead of all buttons.

Reply:
Got it!